frontend/src/containers/Collection/CollectionDetail/CollectionDetail.js

```diff
@@ -101,31 +137,32 @@ class CollectionDetail extends Component {
         return (
             <div className="CollectionDetail">
                 <div className="CollectionDetailContent">
+                    <div className="head">COLLECTION</div>
                     <div className="CollectionInfo">
                         <div id="collectionName">
-                            <h2 id="collectionName">{this.props.thisCollection.name}</h2>
+                            <h2 id="collectionName">{this.state.thisCollection.title}</h2>
                         </div>
                         <div id="collectionInfoMid">
                             <div id="likeStat">
-                                <h5 id="likeCount">{this.props.thisCollection.likesCount}</h5>
+                                <h5 id="likeCount">{this.state.likeCount}</h5>
                                 <h5 id="likeText">Likes</h5>
                             </div>
                             <div id="memberStat">
-                                <h5 id="memberCount">{this.props.thisCollection.members.length}</h5>
+                                <h5 id="memberCount">{this.state.userCount}</h5>
                                 <h5 id="memberText">Members</h5>
                             </div>
                             <div id="collectionButtons">
                                 {likeOrUnlike}
                                 {inviteButton}
-                                {this.props.thisCollection.amIMember ? editButton : <div />}
```

Comment:
Edit Button on Collection Detail page should not be displayed if a user is not a member of this collection or an owner of it. Is it a reason for the change that there is no user related Redux files?

Reply:
If this change is made by a mistake, please commentize the original line for now.

Reply:
Sorry. I thought it would not be required, since there's no features about shared collection. I added it again.